boards/FSM/Test/Src/Test_StateMachine.cpp

```diff
@@ -329,15 +338,26 @@ TEST_F(FsmStateMachineTest, check_brake_pressure_can_signals_in_all_states)
 // FSM-18
 TEST_F(
     FsmStateMachineTest,
-    check_brake_pressure_actuation_can_signals_in_all_states)
+    check_brake_pressure_actuation_can_signal_in_all_states)
 {
-    CheckBinaryStatusCanSignalsInAllStates(
+    CheckBinaryStatusCanSignalInAllStates(
         is_brake_actuated_fake.return_val,
         App_CanTx_GetPeriodicSignal_BRAKE_IS_ACTUATED,
         CANMSGS_FSM_BRAKE_BRAKE_IS_ACTUATED_TRUE_CHOICE,
         CANMSGS_FSM_BRAKE_BRAKE_IS_ACTUATED_FALSE_CHOICE);
 }
 
+TEST_F(
+    FsmStateMachineTest,
+    check_brake_open_or_short_circuited_can_signal_in_all_states)
```

Comment:
Is there a spec we can associate with this test?

Reply:
This seems to be used for debugging. No specs associated to the test

Reply:
Ah, sounds good. Could we leave a quick comment indicating as such?